### PR TITLE
Add Developer Guidelines to doc

### PR DESCRIPTION
**Pull request by phndiaye**
### What does this PR do?

Adds a Developer Guidelines section to the Storybook with a section for tooling

Related https://github.com/upfluence/backlog/issues/421

### What are the observable changes?

Screenshot 2021-03-04 at 11 36 11

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated tests
- [x] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: stories/Guidelines.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    previewTabs: { canvas: { hidden: true } }
+  }}
+/>
+
+# Developer Guidelines
+
+
+Various guidelines to ensure consistency and quality across the Upfluence Frontend Team.
+
+
```
Comment (jlevesy) [inline stories/Guidelines.stories.mdx:16]:
```suggestion
```

File: stories/Guidelines.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  title="Getting Started/Guidelines"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: { canvas: { hidden: true } }
+  }}
+/>
+
+# Developer Guidelines
+
+
```
Comment (jlevesy) [inline stories/Guidelines.stories.mdx:13]:
```suggestion
```

**Review by jlevesy — COMMENTED** [review]
:eyes:

File: stories/Guidelines.stories.mdx (hunk 1 above)
Comment (phndiaye) [reply stories/Guidelines.stories.mdx:16]: Writes a Guideline, doesn't follow it in the Guidelines doc. 😂

Comment (jlevesy) [reply stories/Guidelines.stories.mdx:16]: Do what I say, not what I do :grin:

Comment (phndiaye) [reply stories/Guidelines.stories.mdx]: Exactly hahah

**Review by JulienVannier66 — APPROVED** [review]

**Review by jlevesy — APPROVED** [review]

**Pull request merged**